doc/network-forwards.md

```diff
@@ -1,59 +1,96 @@
 listen\_address  | string     | yes      | IP address to listen on
-description      | string     | no       | Description of Network Forward
-config           | string set | no       | Config key/value pairs (Only `target_address` and `user.*` custom keys supported)
-ports            | port list  | no       | Network forward port list
+description      | string     | no       | Description of the network forward
+config           | string set | no       | Configuration options as key/value pairs (only `target_address` and `user.*` custom keys supported)
+ports            | port list  | no       | List of {ref}`port specifications <network-forwards-port-specifications>`
 
+(network-forwards-listen-addresses)=
 ### Requirements for listen addresses
 
-The listen addresses allowed vary depending on which [network type](#network-types) the forward is associated to.
+The requirements for valid listen addresses vary depending on which network type the forward is associated to.
 
-(network-forwards-bridge)=
-### network: bridge
+Bridge network
+: - Any non-conflicting listen address is allowed.
+  - The listen address must not overlap with a subnet that is in use with another network.
 
-Any non-conflicting listen address is allowed.
+OVN network
+: - Allowed listen addresses must be defined in the uplink network's `ipv{n}.routes` settings or the project's `restricted.networks.subnets` setting (if set).
+  - The listen address must not overlap with a subnet that is in use with another network.
 
-The listen address used cannot overlap with a subnet that is in use with another network.
+(network-forwards-port-specifications)=
+## Configure ports
 
-(network-forwards-ovn)=
-### network: ovn
+You can add port specifications to the network forward to forward traffic from specific ports on the listen address to specific ports on the target address.
+This target address must be different from the default target address.
+It must be within the same subnet as the network that the forward is associated to.
 
-The allowed listen addresses are those that are defined in the uplink network's `ipv{n}.routes` settings, and the project's `restricted.networks.subnets` setting (if set).
+Use the following command to add a port specification:
 
-The listen address used cannot overlap with a subnet that is in use with another network.
+```bash
+lxc network forward port add <network_name> <listen_address> <protocol> <listen_ports> <target_address> [<target_ports>]
+```
 
-## Configure ports
+You can specify a single listen port or a set of ports.
+If you want to forward the traffic to different ports, you have two options:
+
+- Specify a single target port to forward traffic from all listen ports to this target port.
+- Specify a set of target ports with the same number of ports as the listen ports to forward traffic from the first listen port to the first target port, the second listen port to the second target port, and so on.
```

Comment:
Should we give an example or indication as to how these port sets are specified?

Reply:
I don't think we need this here - it's in the properties table below.

Reply:
Ah yes good point.